src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/TlsTests.cs

```diff
@@ -16,21 +16,19 @@
 using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2;
 using Microsoft.AspNetCore.Server.Kestrel.InMemory.FunctionalTests.TestTransport;
 using Microsoft.AspNetCore.Testing;
-using Microsoft.Extensions.Logging.Testing;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.InMemory.FunctionalTests.Http2
 {
     [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "Missing SslStream ALPN support: https://github.com/dotnet/runtime/issues/27727")]
+    [OSSkipCondition(OperatingSystems.Linux, SkipReason = "TLS 1.1 ciphers are now disabled by default: https://github.com/dotnet/docs/issues/20842")]
```

Comment:
Can we move all the skip conditions to the single test in this class? This is a good reason to skip the TlsHandshakeRejectsTlsLessThan12 test, but maybe not other TlsTests. It doesn't matter now, but this might be easy to miss if someone adds more tests to this class.